src/main/java/com/plus/domain/notification/enums/DrawNotificationType.java

```diff
@@ -0,0 +1,14 @@
+package com.plus.domain.notification.enums;
+
+public enum DrawNotificationType {
+	BEFORE_START(1, " 응모 시작 한시간 전 입니다."),
+	BEFORE_END(1, " 응모 종료 한시간 전 입니다.");
+
+	public final Integer time;
+	public final String message;
```

Comment:
```suggestion
	BEFORE_START(" 응모 시작 한시간 전 입니다."),
	BEFORE_END(" 응모 종료 한시간 전 입니다.");
	
	public final String message;
```
String 으로 "한시간 전" 이라고 넣는다면 time은 필요 없을 것 같습니다.

Reply:
지금은 1시간 밖에 없지만 예를 들어 다양한 알림기능이 추가됐을 때를 대비해서 time도 함께 넣었습니다. 알림 스케줄이 실행되는 시간을 계산하려면 time이 필요하거든요!